tests/test_data/test_run_with_example_files_output.txt

```diff
@@ -18,25 +19,28 @@ Dividends:
   NVDA: 0.10 (GBP)
 Interests:
   Charles Schwab: 1.40 (USD)
-Disposal proceeds: £42783.83
+Disposal proceeds: £43798.48
 
-WARNING: Bed and breakfasting for FOO. Disposed on 2021-03-03 and acquired again on 2021-04-02
+WARNING: Bed and breakfasting for VUAG. Disposed on 2020-05-03 and acquired again on 2020-06-02
 
 Second pass completed
 Portfolio at the end of 2020/2021 tax year:
-  GE: 1.00, £8.07
+  VUAG: 30.50, £642.31
+  VNRG: 1.00, £8.98
   NVDA: 1.00, £401.01
-  FOO: 30.50, £543.97
   GOOG: 172.00, £12516.55
+Excess Reported Income:
+  VUAG: £22.26 (included as dividend)
+  VNRG: £0.91 (included as dividend)
 For tax year 2020/2021:
```

Comment:
One last thing, move ERI down to reflect that is part of the current tax year's calculations.
```suggestion
For tax year 2020/2021:
Excess Reported Income:
  VUAG: £22.26 (included as dividend)
  VNRG: £0.91 (included as dividend)
```